### ISSUE TITLE
[Feature Request] Checkpoint loader with info

### ISSUE THREAD

**Issue by wouterdebie** [rgthree/rgthree-comfy#262]
I absolutely love the power lora loader and use the information feature quite a bit to look up settings. Having similar functionality for checkpoints would be great if possible.

**Comment by gisu** [comment]
Me too. I like the way Power Lora presents the information. :)

Maybe another idea if you could extend the loader. Could you extend it so that you can store a txt or json file info about the checkpoints like CFG, Steps, Sampler, Scheduler and Clip. The loader will then search with the model or filename and output the information about the outputs when it finds a match. This can then be used to feed the inputs of the sampler and saves you having to search for the optimum settings.

**Comment by pasfwyt4p8** [comment]
Agreed. I would LOVE to see this. My #1 request. The power lora loader is my favorite node because of the SHOW INFO. This would eliminate the need for me to make all these notes—copying and pasting text from Civitai.com, going back and forth reminding myself what this checkpoint's recommended settings are, etc. etc.
